index.js

```diff
@@ -273,83 +301,47 @@ MyQ2Platform.prototype.getDevice = function (callback) {
 
       for (var i = 0; i < devices.length; i++) {
         var device = devices[i];
-        var deviceType = device.MyQDeviceTypeId;
-        var deviceDesc = "Unknown";
-
-        // Search for specific device type
-        if(deviceType != 1) continue;
-
-        for (var j = 0; j < device.Attributes.length; j ++) {
-          var thisAttributeSet = device.Attributes[j];
-          // Search for device name
-          if(thisAttributeSet.AttributeDisplayName === "desc") {
-            deviceDesc = thisAttributeSet.Value;
-          }
-        }
+        var deviceType = device.device_type;
+        var deviceDesc = device.name;
 
         // Is this gateway one of the specified gateways in the config
-        gatewaysKeyed[device.MyQDeviceId] = deviceDesc;
-        if(self.gateways.indexOf(deviceDesc) > -1 || self.gateways.indexOf(device.MyQDeviceId) > -1) allowedGateways.push(device.MyQDeviceId);
+        gatewaysKeyed[device.serial_number] = deviceDesc;
+        if(self.gateways.indexOf(deviceDesc) > -1 || self.gateways.indexOf(device.serial_number) > -1) allowedGateways.push(device.serial_number);
       }
 
       // Look through the array of devices for all the openers
       for (var i = 0; i < devices.length; i++) {
         var device = devices[i];
-        var deviceType = device.MyQDeviceTypeName;
+        var deviceType = device.device_type;
 
         // Search for specific device type
-        if(deviceType === "Garage Door Opener WGDO" || deviceType === "GarageDoorOpener" || deviceType === "VGDO" || deviceType === "Gate") {
-          var thisDeviceID = device.MyQDeviceId.toString();
-          var thisSerial = device.SerialNumber.toString();
+        if(deviceType == 'virtualgaragedooropener' || deviceType === "Garage Door Opener WGDO" || deviceType === "GarageDoorOpener" || deviceType === "VGDO" || deviceType === "Gate") {
```

Comment:
when testing, had to add `|| deviceType == "garagedooropener"` to get this working with recently added liftmaster

error prior to adding:
`[7/3/2020, 5:07:18 PM] [MyQ2] Error: Couldn't find a MyQ door device.`

Reply:
Cheers, I added that. Yea I only have one device to test with =)

Reply:
I'm still getting `Error: Couldn't find a MyQ door device.` so there must be another deviceType that we need. I'll check this out tomorrow and grab the name of my device so we can add it here. Thanks again!

Reply:
Hopefully by tomorrow that wont be necessary one way or the other =)

FYI if you enable `debug` logging on this plugin there would be some log output that might be helpful to figure out your issue.